app/src/main/java/org/simple/clinic/home/overdue/OverdueScreenController.kt

```diff
@@ -18,7 +24,26 @@ class OverdueScreenController @Inject constructor() : ObservableTransformer<UiEv
   }
 
   private fun screenSetup(events: Observable<UiEvent>): Observable<UiChange> {
-    return events.ofType<OverdueScreenCreated>()
-        .map { { ui: Ui -> ui.updateOverdueList() } }
+    return events
+        .ofType<OverdueScreenCreated>()
+        .flatMap { appointmentRepo.appointments() }
+        .map { appointments ->
+          appointments.map {
+            OverdueListItem(
+                it.id,
+                it.patientId.toString(),
+                "Female",
+                22,
+                175,
+                55,
+                30,
+                getOverdueDays(it.date))
```

Comment:
Can you use named parameters? They look neater and do not get affected by changes in the order of parameters.

Reply:
Hrm, I just realized that it's possible you have parameter hints turned on in your IDE. Considering that the code reviewer cannot see them, they're not very useful.

Reply:
Do you also want to use `it` instead of `appointments` and make your Rx chain slightly neater?